src/test/transaction_tests.cpp

```diff
@@ -314,7 +315,6 @@ BOOST_AUTO_TEST_CASE(test_Get)
 
 BOOST_AUTO_TEST_CASE(test_IsStandard)
 {
-    LOCK(cs_main);
```

Comment:
In an older version I believe that allowed me to remove the main include, but since that's not the case anymore I guess I should leave this out.